Formula/jq.rb

```diff
@@ -21,8 +21,8 @@ class Jq < Formula
     depends_on "libtool" => :build
   end
 
-  depends_on "oniguruma"  # jq depends > 1.5
-  depends_on "bison" => :build # jq depends on bison > 2.5
+  depends_on "oniguruma" # jq depends > 1.5
+  depends_on "bison" => :build # jq depends on bison >= 3.0
```

Comment:
`bison` dependency can be removed if you could add `--disable-maintainer-mode` to configure options.

```rb
    system "./configure", "--disable-dependency-tracking",
                          "--disable-silent-rules",
                          "--disable-maintainer-mode",
                          "--prefix=#{prefix}"
```